_posts/2022-09-07-quick-tips-to-speed-up-ci.markdown

```diff
@@ -0,0 +1,145 @@
+---
+layout: epic
+title: "Parallelizing Jest and Cypress.io Tests on CircleCI"
+subtitle: A few simple tips on how to speed up your CI
+date: 2022-09-07
+categories: [CircleCI, CI, Tests, Jest, Cypress]
+author: [chris]
+---
+
+At Artsy, exploring ways to improve the developer experience is part of our
+makeup. Whether it's implementing [hot-swapping][hot-swap] for Express.js or
+integrating the Rust-based [SWC compiler][swc] into our front-end build
+pipeline, we're always trying to reduce the amount of time it takes for a code
+cycle to take place. CI is no exception. When a developer opens a PR, we want to
+ensure they get timely feedback. Have their unit tests passed? Does the app
+build correctly? And how about smoke tests? Each of these jobs are complex
+processes that take time, and the more one can parallelize said tasks the less
+waiting devs will need to do. Scaled out to a whole engineering org, minor
```

Comment:
I'd phrase the latter part '...the less devs will need to wait.' instead of '...the less
waiting devs will need to do.'